9.2/paper.tex

```diff
@@ -446,7 +446,61 @@ \subsection{Improved large-scale performance}
 \subsection{Better support for parallel $hp$-adaptive algorithms}
 \label{subsec:hp}
 
-\todo[inline]{Marc: Your section}
+With the previous release, \dealii supports $hp$-adaptive finite element methods
+on distributed memory systems \cite{dealII91}. We implemented the bare functionality
+for $hp$-adaptive methods with the objective to offer the greatest flexibility in
+their application. Here, reference finite elements still had to be assigned manually
+to each cell, which may not lead to an optimal mesh and is tedious.
+
+With the current release, $hp$-adaptive finite element methods have been further
+expanded in both the range of features in terms of decision strategies and their 
```

Comment:
```suggestion
expanded in both the range of features like decision strategies and their 
```
? Here I'm not sure what you want to say.

Reply:
I made two sentences out of this one. Hope that helps :)